src/formatDate.js

```diff
@@ -8,7 +8,71 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
```

Comment:
The function `formatDate` is well-named and follows the task description. However, it could benefit from some refactoring for improved readability and maintainability.